bench-tps/src/send_batch.rs

```diff
@@ -402,30 +435,68 @@ impl<'a> CreateNonceTransactions<'a> for NonceContainer<'a> {
         };
         self.send_transactions(client, nonce_rent, log_progress);
     }
+}
 
-    fn make(&mut self, nonce_rent: u64, to_fund: &'a NonceChunk<'a>) {
-        let mut make_txs = Measure::start("make_txs");
-        let to_fund_txs: NonceContainer = to_fund
-            .par_iter()
-            .map(|(authority, nonce)| {
-                let instructions = system_instruction::create_nonce_account(
-                    &authority.pubkey(),
-                    &nonce.pubkey(),
-                    &authority.pubkey(),
-                    nonce_rent,
-                );
+struct NonceWithdrawSigners<'a>(&'a Keypair, &'a Keypair);
```

Comment:
nit: It would be clearer for the second parameter to be a `Pubkey` since it's not signing, but I think that'll make things more complicated in the implementation, like `(&'a Keypair, Pubkey)`, which is probably worse.  I imagine you already considered this tradeoff :sweat_smile:

Reply:
Actually this is really good point! And it happens that it is quite easy to implement